GetDKAN/dkan
# Add new curl function to limit how much of the file is returned

Connects https://github.com/NuCivic/healthdata/issues/1062
## Recap
When running `$info = $this->getFileInfoHelper($url, FALSE)` the content of the file is copied in full as `curl_setopt($ch, CURLOPT_RANGE, '0-1000'); ` is not honored.
This causes out of memory errors when harvesting.

Creating a second helper function to run if `$info = $this->getFileInfoHelper($url, TRUE)` does not return the headers we need.
 
## QA Steps
- Create harvest source from https://data.cdc.gov/data.json
- run `drush dkan-h data_cdc_gov`
- confirm the harvest finishes AND the resource files are correctly saved as remote files and not as an api
